fs/info.py

```diff
@@ -58,38 +58,13 @@ def __eq__(self, other):
         # type: (object) -> bool
         return self.raw == getattr(other, 'raw', None)
 
-    @typing.overload
-    def _make_datetime(self, t):
-        # type: (None) -> None
-        pass
-
-    @typing.overload
-    def _make_datetime(self, t):
-        # type: (int) -> datetime
-        pass
-
     def _make_datetime(self, t):
         # type: (Optional[int]) -> Optional[datetime]
         if t is not None:
             return self._to_datetime(t)
         else:
             return None
 
-    @typing.overload
-    def get(self, namespace, key, default=None):
-        # type: (Text, Text, Optional[T]) -> Optional[T]
-        pass
-
-    @typing.overload
-    def get(self, namespace, key):
-        # type: (Text, Text) -> Optional[T]
-        pass
-
-    @typing.overload
-    def get(self, namespace, key, default):
-        # type: (Text, Text, T) -> T
-        pass
-
```

Comment:
Just as there is a `_fscompat` module, maybe we could have a `_tycompat` where `overload` is defined as a no-op decorator when it cannot be imported from the `typing` module ? 

This is part of the public API, and I think it's a pity to remove a finer typed API just because of a single Python version.